.github/workflows/lint.workflow.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  push:
+    branches-ignore:    
+      - master
+      - gh-pages
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Node

```

Comment:
[Comment]: Should be after `Checkout repo` I believe.

Reply:
Went ahead and fixed that in this latest commit 👍